xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-table.html

```diff
@@ -0,0 +1,90 @@
+<template>
+    <table class="xwiki-livetable">
+
```

Comment:
Totally unwanted noob remark: has ``vue-good-table`` been considered for the table layout?

Reply:
Indeed we considered using existing table plugins at first, but as we are doing most of the operations in the server (filtering, sorting, pagination, ...), the table only receives a list of the rows corresponding to this configuration. So we can't use the common functionalities of the plugin as it would do it on the client side with incomplete data, so at the end it's no longer worth it.